thirdparty/proto/v1/ipns/ipns_entry.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package sonrio.sonr.thirdparty.ipns;
+
+option go_package = "github.com/sonr-io/sonr/thirdparty/proto/v1/ipns";
+
+message IpnsEntry {
+  enum ValidityType {
+    // setting an EOL says "this record is valid until..."
+    EOL = 0;
+  }
+  required bytes value = 1;
```

Comment:
Everything is optional by default in proto3 and it's considered bad practice to have required fields.

https://stackoverflow.com/a/31814967/4092920